### PR TITLE
Feature/add recipe page

### PR DESCRIPTION
**Pull request by shivams10**

### CODE REVIEW

File: src/pages/recipe/index.jsx [hunk 1]
```diff
@@ -1,9 +1,104 @@
...
+  };
+
+  if (modal) {
+    document.body.classList.add("active-modal");
+  } else {
+    document.body.classList.remove("active-modal");
+  }
+
+  const search = () => {
+    searchForRecipe(inputRef.current.value);
```
Comment (siddhi-rajput-22) [inline src/pages/recipe/index.jsx]: useRef is not required here. Use another way to get the input value of search field

File: src/App.js [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
 import Navbar from "./components/navbar";
 import "./App.css";
 
 function App() {
   const [navVisible, showNavbar] = useState(false);
 
-  return <Navbar visible={navVisible} show={showNavbar} size={cart.length} />;
+  return (
+    <>
+      <Navbar visible={navVisible} show={showNavbar} size={cart.length} />
```
Comment (siddhi-rajput-22) [inline src/App.js:12]: where is cart defined ?

Comment (shivams10) [reply src/App.js:12]: actually it was defined in another component i will push it in next pull request

File: src/pages/recipe/index.jsx (hunk 1 above)
Comment (shivams10) [reply src/pages/recipe/index.jsx]: okay i am doing it.

Comment (shivams10) [reply src/pages/recipe/index.jsx]: image
Done

File: src/pages/recipe/index.jsx [hunk 3]
```diff
@@ -22,11 +22,11 @@ const Recipe = () => {
     document.body.classList.remove("active-modal");
   }
 
-  const search = () => {
-    searchForRecipe(inputRef.current.value);
-    inputRef.current.value = "";
+  const search = (searchValue) => {
```
Comment (siddhi-rajput-22) [inline src/pages/recipe/index.jsx]: change the function name to 'searchRecipe'. Always be specific regarding the functions names, it should tell what function does.

Comment (shivams10) [reply src/pages/recipe/index.jsx]: okay!

Comment (shivams10) [reply src/pages/recipe/index.jsx]: image
Done!

**Pull request merged**